app/assets/javascripts/preload-application-data.js

```diff
@@ -39,4 +39,26 @@
     Discourse.S3CDN = setupData.s3Cdn;
     Discourse.S3BaseUrl = setupData.s3BaseUrl;
   }
+
+  Ember.RSVP.configure("onerror", function(e) {
+    // Ignore TransitionAborted exceptions that bubble up
+    if (e && e.message === "TransitionAborted") {
+      return;
+    }
+
+    if (Discourse.Environment === "development") {
```

Comment:
I moved this here because this is the earliest place that we have access to the initialized `Discourse.Environment`.